packages/grid/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -304,9 +304,9 @@ function defaultPasteResolver({
   });
 }
 
-function isPasteShortcut(event: React.KeyboardEvent) {
-  return (event.ctrlKey || event.metaKey) && event.key === 'v';
-}
+const isPasteShortcut = (event: React.KeyboardEvent) => {
+  return (event.ctrlKey || event.metaKey) && event.key.toLowerCase() === 'v';
+};
```

Comment:
While we're touching this, if you want to add logic to avoid matching `ctrl-shift-v` and `ctrl-alt-v`, that would be nice.

Reply:
done